cms/templatetags/cms_tags.py

```diff
@@ -1058,7 +1064,10 @@ def render_tag(self, context, **kwargs):
         context.push()
         template = self.get_template(context, **kwargs)
         data = self.get_context(context, **kwargs)
-        data['content'] = mark_safe(kwargs['nodelist'].render(data))
+        if get_cms_setting('UNESCAPED_RENDER_MODEL_TAGS'):
```

Comment:
@mkoistinen have you considered encapsulating this if to make the code more maintenable?